docs/experiment/experiment_creation.md

```diff
@@ -0,0 +1,75 @@
+# Experiment Creation
+
+Start by [navigating](../project/project_introduction.md#project-navigation) 
+to the project summary view of your project of interest.
+To create an experiment in the selected project, click the add button within the experiment component on the top right.
+![project_summary](../project/images/project_summary.png)
+
+!!! info "Project role"
+    Should you not see the add button,
+    ensure that you have been granted the "write" or "admin" project role by the project owner/admin!
+
+This will open the create experiment dialog, in which you can specify the name of the experiment and the included species, specimen and analyte
+information of the involved organisms.
+To provide these details you can select one or more entries via the provided combo boxes.
+Start by providing an easily identifiable name and at least 2 letters of your species, specimen or analyte in their respective inputs
+fields and possible selection options will appear.
+![experimental_information](images/create_experiment_search.png)
+
+!!! info "Ontology id"
+    Behind each selectable option within the species, specimen and analyte input fields
+    , the unique ontology identifier from one of several ontologies is stored.
+
+Optionally you can also select one of the provided icons most fitting for your selected species and specimen.
+
+Once all the required information has been provided you can create your experiment via the "Add"
+button below.
+![experimental_information](images/create_experiment.png)
+
+
+## Experimental Variable Creation
+
+Experimental variables can be considered as the building blocks for the condition with which the samples in your experiments were treated.  
+One or more of these variables can be added in the experiment summary screen by selecting the experimental variables tab and pressing the "add variables" button within the disclaimer.
+![experiment_summary](images/experimental_summary_no_variables.png)
+
+!!! info "Project role"
+    Should you not see the add button,
+    ensure that you have been granted the "write" or "admin" project role by the project owner/admin!
+
+Once clicked the add variables dialog will appear, in which one or more variables can be defined and linked to your experiment.
+![add_experimental_variables](images/add_experimental_variables.png)
```

Comment:
NIT: I don't know if these examples are fitting. How about treatment/control and wildtype/mutant?

Reply:
I'll keep that in mind for the next round of documentation before the "real" workshop in july 👍